packages/js/src/components/contentAnalysis/ReadabilityAnalysis.js

```diff
@@ -156,7 +155,7 @@ class ReadabilityAnalysis extends Component {
 
 		// The reason of adding the check if Elementor is active or not is because `isBlockEditor` method also returns `true` for Elementor.
 		// The reason of adding the check if the Elementor editor is active, is to stop showing the buttons in the in-between screen.
-		return hasAIFixes && isBlockEditor() && isNotElementorPage && (
+		return hasAIFixes && isNotElementorPage && (
```

Comment:
Please don't forget to update the documentation here (and in `SEOAnalysis.js` as well) 👍